package-lock.json

```diff
@@ -649,6 +649,21 @@
                 "concat-map": "0.0.1"
             }
         },
+        "node_modules/@eslint/eslintrc/node_modules/globals": {
+            "version": "13.24.0",
```

Comment:
This being duplicated was one reason I avoided upgrading this, but it's not harmful and eventually we'll get ESLint v9 and this all will drop away.